infra/app/env-template/main.tf

```diff
@@ -116,18 +164,40 @@ module "staff" {
   service_cluster_arn  = module.service_cluster.service_cluster_arn
   container_port       = 3000
   enable_exec          = var.staff_enable_exec
+  enable_healthcheck   = false
   container_secrets = [
     {
       name      = "LOWDEFY_SECRET_PG_CONNECTION_STRING",
-      valueFrom = "arn:aws:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter${module.participant_database.admin_db_url_secret_name}"
+      valueFrom = "arn:aws:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter${module.participant_database.admin_db_url_secret_name}",
+    },
+    {
+      name      = "LOWDEFY_SECRET_OPENID_CLIENT_ID",
+      valueFrom = "arn:aws:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter${module.staff_idp.client_id_secret_name}",
+    },
+    {
+      name      = "LOWDEFY_SECRET_OPENID_CLIENT_SECRET",
+      valueFrom = "arn:aws:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter${module.staff_idp.client_secret_secret_name}",
```

Comment:
Add @TODO to change this. Our outputs should just be the arns! They don't need to be the name of the secret itself